kitsune/search/admin.py

```diff
@@ -48,6 +38,33 @@ class DeleteError(Exception):
     pass
 
 
+def create_batch_id():
+    """Returns a batch_id"""
+    # TODO: This is silly, but it's a good enough way to distinguish
+    # between batches by looking at a Record. This is just over the
+    # number of seconds in a day.
+    return str(int(time.time()))[-6:]
+
+
+def handle_reconcile(request):
+    """Reconcile all the things"""
+    outstanding = Record.objects.outstanding().count()
+    if outstanding > 0:
+        raise ReindexError('There are %s outstanding chunks.' % outstanding)
+
+    batch_id = create_batch_id()
+
+    for cls, indexable in get_indexable():
+        index = cls.get_index()
+        doc_type = cls.get_mapping_type_name()
+        chunk_name = 'Reconciling: %s' % doc_type
+        rec = Record(batch_id=batch_id, name=chunk_name)
+        rec.save()
```

Comment:
Any reason not to use `Record.objects.create(batch_id=batch_id, name=chunk_name)` instead of the two line version?